dataland-frontend/src/components/forms/parts/fields/ProductionSitesFormField.vue

```diff
@@ -34,19 +41,12 @@ import UploadFormHeader from "@/components/forms/parts/elements/basic/UploadForm
 import { FormFieldProps } from "@/components/forms/parts/fields/FormFieldProps";
 
 export default defineComponent({
-  name: "ProductionSiteFormField",
+  name: "ProductionSitesFormField",
   props: FormFieldProps,
   data() {
     return {
-      productionSite: Object as LksgProductionSite,
-      listOfGoodsOrServicesString: "",
-      listOfProductionSites: [
-        {
-          id: 0,
-          listOfGoodsOrServices: [] as string[],
-          listOfGoodsOrServicesString: "",
-        },
-      ],
+      existingProductionSites: [] as LksgProductionSite[],
+      listOfProductionSiteIds: [0] as number[],
```

Comment:
Why do we need to initialize this list with "0" as already existing production site id?

Reply:
Got it, thx.